cpp/velox/benchmarks/GenericBenchmark.cc

```diff
@@ -126,7 +127,9 @@ auto BM_Generic = [](::benchmark::State& state,
   auto memoryManager = getDefaultMemoryManager();
   auto runtime = Runtime::create(kVeloxRuntimeKind, conf);
   const auto& filePath = substraitJsonFile;
-  auto plan = getPlanFromFile(filePath);
+  auto plan = getPlanFromFile("Plan", filePath);
+  const auto& splitPath = splitFile;
```

Comment:
Why do we need this variable, can we use splitFile instead?

Reply:
just follow `filePath`, will remove.